McBopomofoTests/KeyHandlerBopomofoTests.swift

```diff
@@ -1,3 +1,26 @@
+// Copyright (c) 2011 and onwards The McBopomofo Authors.
```

Comment:
2022 for files created this year? Same for other files here.

Reply:
OK. Let me update it.